tests/functional-all.js

```diff
@@ -16,6 +16,7 @@ define([
   "./functional/comments-non-auth.js",
   "./functional/contributors-non-auth.js",
   "./functional/comments-auth.js",
+  "./functional/image-uploads-auth.js",
```

Comment:
Maybe, instead of renaming the file I just put this test inside of comments-auth.js. What do you think?